.github/workflows/trivy.yaml

```diff
@@ -20,6 +20,7 @@ jobs:
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@0.9.1
```

Comment:
As mentioned, I think we should track `master` here.

Reply:
To add to @wadells's comment, I would prefer auto-merging Dependabot updates vs tracking the master branch.  But currently hitting the merge button is not a significant overhead.

Reply:
I'm ambivalent, as this scanner pulls down a vulnerability database every time it runs. It seems to me like this is more about updating the action and test-driving code.